src/Framework/TestCase.php

```diff
@@ -174,6 +179,10 @@ protected function tearDown()
 	{
 	}
 
+	/** @internal */
+	public function failsafeTearDown() {
```

Comment:
The only point of this method is to make the stuff PHP 5.3 compatible - make possible to call tearDown from a callback, as PHP 5.3 doesn't see "$this" context from anonymous function.